dev/com.ibm.ws.security.oidc.server_fat.claimPropagation/fat/src/com/ibm/ws/security/openidconnect/server/fat/claimPropagation/CommonTests/BasicIdTokenClaimPropagationTests.java

```diff
@@ -0,0 +1,766 @@
+     * The set<...>Expectation methods that are called will set their specific expectations appropriately
+     */
+
+    /**
+     * The Third party OP will populate the idToken that it creates with the groupIds claim (by default)
+     * The Third party OP does not populate the idToken that it creates with the any extra claims
+     * The intermediate OP does not have thirdPartyIDTokenClaims configured.
+     * No third party OP claims will be included in the idToken produced by the intermediate OP.
+     *
+     * @throws Exception
+     */
+    @Mode(TestMode.LITE)
+    @Test
+    public void ThirdPartyIDTokenClaims_none_3rdPartyDoesNotPropagate() throws Exception {
+
+        String appName = "noExtraClaims_3rdPartyDoesNotPropagate";
+        WebConversation wc = new WebConversation();
+        TestSettings updatedTestSettings = testSettings.copyTestSettings();
+        updatedTestSettings.setTestURL(testSettings.getTestURL().replace("SimpleServlet", "simple/" + appName));
+        setParmValues(updatedTestSettings.getUserName());
+
+        List<validationData> expectations = vData.addSuccessStatusCodes(null);
+        expectations = setGroupIdsExpectations(expectations, thirdPartyPropagatedIdTokenFalse);
+        expectations = setTestParmExpectations(expectations, noThirdPartyParm, thirdPartyPropagatedIdTokenFalse);
+        expectations = setRealmNameExpectations(expectations);
+        expectations = setIssExpectations(expectations, appName);
+        expectations = setAudExpectations(expectations, appName);
+
+        genericRP(_testName, wc, updatedTestSettings, steps, expectations);
+    }
+
+    /**
+     * The Third party OP will populate the idToken that it creates with the groupIds claim (by default)
+     * The Third party OP does not populate the idToken that it creates with the any extra claims
+     * The intermediate OP does have thirdPartyIDTokenClaims configured with "groupIds".
+     * Only the third party OP groupIds claim will be included in the idToken produced by the intermediate OP.
+     *
+     * @throws Exception
+     */
+    @Mode(TestMode.LITE)
+    @Test
+    public void ThirdPartyIDTokenClaims_groupIds_3rdPartyDoesNotPropagate() throws Exception {
+
+        String appName = "propagateGroupIdsIdTokenClaims_3rdPartyDoesNotPropagate";
+        WebConversation wc = new WebConversation();
+        TestSettings updatedTestSettings = testSettings.copyTestSettings();
+        updatedTestSettings.setTestURL(testSettings.getTestURL().replace("SimpleServlet", "simple/" + appName));
+        setParmValues(updatedTestSettings.getUserName());
+
+        List<validationData> expectations = vData.addSuccessStatusCodes(null);
+        expectations = setGroupIdsExpectations(expectations, thirdPartyPropagatedIdTokenTrue);
+        expectations = setTestParmExpectations(expectations, allUniqueIntermedParms, thirdPartyPropagatedIdTokenFalse);
```

Comment:
I'm confused - is `thirdPartyPropagatedIdTokenTrue` or `thirdPartyPropagatedIdTokenFalse` supposed to be used here?

Reply:
created:
    protected static final Boolean thirdPartyPropagatedTestClaimsInIdTokenTrue = true;
    protected static final Boolean thirdPartyPropagatedTestClaimsInIdTokenFalse = false;
    protected static final Boolean thirdPartyPropagatedGroupIdsInIdTokenTrue = true;
    protected static final Boolean thirdPartyPropagatedGroupIdsInIdTokenFalse = false;
and will use those in lieu of more generic thirdPartyPropagatedIdTokenTrue and thirdPartyPropagatedIdTokenFalse